### PR TITLE
LPD-17467 Logic for state flow implemented

### PR DESCRIPTION
**Pull request by victorKammerer**
Hey, @rijema!! I just checked the solution code for LPD-17467 and apparently it's working fine, could you double check it for me?

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

**Comment by victorKammerer** [comment]
All analysis were done here, all tests covered and passed.

File: modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/test-object-definition-state-manager.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
+{
+	"accountEntryRestricted": false,
+	"accountEntryRestrictedObjectFieldName": "",
+	"actions": {
```
Comment (rijema) [inline modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/test-object-definition-state-manager.json:4]: Hi @victorKammerer , I believe this document will be restructured soon, that's why we're stuck for now. I'm closing this PR and will be analyzing it through the liferay-object repo. Thank you and congrats for your first analysis! 
image

**Comment by rijema** [comment]
Following PR: https://github.com/liferay-objects/liferay-portal/pull/3340

**Pull request closed without merging**